### PR TITLE
Add Arc-Dark theme

### PR DESCRIPTION
**Pull request by Hanouta**
This adds a theme option which is based on the Arc-Dark theme. It's a pretty popular theme, especially useful for people who don't want to use a light theme but find the other Dark theme options to be too dark.

### CODE REVIEW

**Comment by alainm23** [comment]
I like the theme.

image

File: src/Utils.vala [hunk 1]
```diff
@@ -740,6 +740,17 @@ public class Utils : GLib.Object {
...
+                Gtk.Settings.get_default ().gtk_application_prefer_dark_theme = true;
+            } else if (appearance_mode == 3) {
+                base_color = "#353945";
+                check_border_color = "grey";
+                projectview_color = "#404552";
+                pane_color = "#353945";
+                pane_selected_color = "#2B303B";
+                pane_text_color = "#fefeff";
+                popover_background = "#353945";
+                row_selected_color = "shade (#fefeff, 0.25)";
```
Comment (alainm23) [inline src/Utils.vala]: I think we should improve the border color of the task, maybe a deeper black.
image

**Review by alainm23 — CHANGES_REQUESTED** [review]

File: src/Utils.vala (hunk 1 above)
Comment (Hanouta) [reply src/Utils.vala]: Just pushed an update, do you like the new one better?

**Comment by alainm23** [comment]
Thank you very much, it looks better now.

**Pull request merged**